README.md

```diff
@@ -40,41 +45,40 @@ and the Python API documentation.
 * Can quickly [access cells, surfaces, and materials by their numbers](https://www.montepy.org/starting.html#collections-are-accessible-by-number). For example: `cell = problem.cells[105]`.
 * Can quickly update cell parameters, [such as importances](https://www.montepy.org/starting.html#setting-cell-importances). For example `cell.importance.neutron = 2.0`.
 * Can easily [create universes, and fill other cells with universes](https://www.montepy.org/starting.html#universes).
-* Currently has over 430 test cases.
+* Currently has over 550 test cases.
 
  
 Here is a quick example showing multiple tasks in MontePy:
 
 ```python
-import montepy
-# read in file
-problem = montepy.read_input("foo.imcnp")
+>>> import montepy
```

Comment:
If the workflow of this example is in a python script I would recommend removing the `>>>` and `...` so folks can copy paste into files more easily

Reply:
Right, that's a good point. This formatting is necessary for the code to be recognized by `doctest` to ensure it's tested. Though there are some viable alternatives possibly that I need to look into further. 

`ipython` is able to understand this notation, but obviously that wouldn't work for dropping this in a `.py` file.

Reply:
I found a better library, so this is now unnecessary.